sexysolver/sexysolver.h

```diff
@@ -0,0 +1,334 @@
+/*  SexySolver, SexySolver Intenal Library developed by Robert Lancaster, 2020
```

Comment:
Yes that is correct, and yes that is a good idea.  I hadn't gotten as far as detailed documentation yet, primarily because before now, it was changing very rapidly, any documentation that I wrote would have been quickly outdated.  I also was just  really focused on just getting it all working nicely.